metrics/cloudwatch/cloudwatch_test.go

```diff
@@ -193,4 +193,54 @@ func TestHistogram(t *testing.T) {
 	if err := svc.testDimensions(n99, label, value); err != nil {
 		t.Fatal(err)
 	}
+
+	// now test with only 2 custom percentiles
+	//
+	svc = newMockCloudWatch()
+	cw = New(namespace, svc, WithLogger(log.NewNopLogger()), WithPercentiles(0.50, 0.90))
+	histogram = cw.NewHistogram(name).With(label, value)
+
+	// note we are 'abusing' the p50 and p90 return values
+	// as these are hardcoded in the teststat.TestHistogram framework
+	customQuantiles := func() (p50, p90, p95, p99 float64) {
+		err := cw.Send()
+		if err != nil {
+			t.Fatal(err)
+		}
+		svc.mtx.RLock()
+		defer svc.mtx.RUnlock()
+		p50 = svc.valuesReceived[n50]
+		p90 = svc.valuesReceived[n90]
+
+		// our teststat.TestHistogram wants us to give p95 and p99,
+		// but with custom percentiles we don't have those.
+		// So fake them. Maybe we should make teststat.nvq() public and use that?
+		p95 = 541.121341
+		p99 = 558.158697
```

Comment:
Thanks for you comment @peterbourgon ; Do you mean leave out the whole added part about testing WithPercentile() ? 
I think that's the only way. 

We can also leave it hardcoded for now; it's all fine with me. I actually do think it's useful to test the WithPercentile(), but I'm not really attached to it. 

If you have a preference, let me know; otherwise I'll leave it as-is.